### PR TITLE
Include `userId` in the `MatomoTracker.initialize` params

### PR DESCRIPTION
**Pull request by juliobetta**

### CODE REVIEW

File: packages/js/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Before you're able to use this Matomo Tracker you need to initialize Matomo with
 const MatomoInstance = new window.MatomoTracker({
   urlBase: 'https://LINK.TO.DOMAIN',
   siteId: 3, // optional, default value: `1`
+  userId: '', // optional, default value: `undefined`
```
Comment (chrisvanmook) [inline packages/js/README.md:44]: A bit nitpicky, but could you provide an example of an ID, like `UID76903202` instead of an empty string?

**Comment by chrisvanmook** [comment]
@juliobetta Thanks for your collaboration! Could you rebase your branch on master and re-generate `bundle.min.js` please?

**Comment by juliobetta** [comment]
> @juliobetta Thanks for your collaboration! Could you rebase your branch on master and re-generate `bundle.min.js` please?

sure! ... and done! 😉

**Comment by chrisvanmook** [comment]
@juliobetta I've published v0.1.5 including your changes

**Pull request merged**